Python/Product/Analysis/Intellisense/AnalysisQueue.cs

```diff
@@ -145,10 +174,15 @@ sealed class AnalysisQueue : IDisposable {
             }
         }
 
+        public void WaitForActivity(int millisecondsTimeout) {
```

Comment:
Method isn't used anywhere, and as a result, `_activityEvent` isn't used as well.